coalib/settings/ConfigurationGathering.py

```diff
@@ -179,13 +182,16 @@ def warn_nonexistent_targets(targets, sections, log_printer=None):
 
     # Can't be summarized as python will evaluate conditions lazily, those
     # functions have intended side effects though.
-    files_config_absent = warn_config_absent(sections, 'files')
-    bears_config_absent = warn_config_absent(sections, ['bears', 'aspects'])
-    if files_config_absent or bears_config_absent:
+    files_config_absent = warn_config_absent(sections, 'files',
+                                             only_global_bears)
```

Comment:
maybe do
```
if not only_global_bears:
    warn_config_absent(sections, 'files)
```
Then we save the `only_global_bears` param for that function.

Reply:
@Makman2 but how will `warn_config_absent` know that only global bears are being run?? we need to suppress the warning message in `warn_config_absent` if `only_global_bears` is true!

Reply:
In case you don't want me to pass around `only_global_bears`

Reply:
but we don't have to call `warn_config_absent` in case `only_global_bears` is `True`, or not? I guess I'm missing something ;)

Reply:
now I get what you mean, in that case I can remove the condition inside the function `warn_config_absent`

Reply:
btw I don't think its making any difference 😶

Reply:
sorry, that way I won't have to pass around `only_global_bears` 👍